draggable.js

```diff
@@ -5,54 +5,83 @@
   else this[moduleName] = definition();
 
 })('draggable', function definition() {
-  function addEventListener(element, eventName, handler) {
-                if (element.addEventListener) {
-                    element.addEventListener(eventName, handler, false);
-                } else if (element.attachEvent) {
-                    element.attachEvent('on' + eventName, handler);
+  var addEventListener = ( function (w) {
+                if (w.addEventListener) {
+                    return function (element, eventName, handler){
+                      element.addEventListener(eventName, handler, false);
+                    }
+                } else if (w.attachEvent) {
```

Comment:
Unless draggable.js need to support IE8 and below - we can remove the attachEvent stuff. What do you say @gtramontina?